main.nf

```diff
@@ -354,6 +352,25 @@ workflow {
     )
 }
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    FUNCTIONS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+//
+// Get attribute from genome config file e.g. fasta
+//
+
+def getGenomeAttribute(attribute) {
```

Comment:
but then we also got `getGenomeAttribute` defined [here](https://github.com/maxulysse/nf-core_sarek/blob/cbb2a62180d6a01f894acfd4ade04ae8735da22f/subworkflows/local/utils_nfcore_sarek_pipeline/main.nf#L258).  That seems odd 🤔